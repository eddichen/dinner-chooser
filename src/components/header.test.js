import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import { theme, GlobalStyle } from '../theme';
import Header from './header';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <ThemeProvider theme={theme}>
          <GlobalStyle />
          <Header siteTitle="Default Starter" />
        </ThemeProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
