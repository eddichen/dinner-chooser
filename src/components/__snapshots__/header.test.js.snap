// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="sc-AxjAm hXdFMb"
>
  <div
    className="sc-AxirZ kpObpS"
  >
    <h1
      className="sc-AxiKw iXhYZV"
    >
      <a
        className="sc-AxhCb gDcsxK"
        href="/"
      >
        Default Starter
      </a>
    </h1>
  </div>
</header>
`;
